src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/update/actions/UpdateFlowAction.java

```diff
@@ -82,8 +82,7 @@ public UpdateFlowAction(PersistenceManager persistenceManager, int transactionRe
     }
 
     private RequestedFlow updateFlow(Flow flow, RequestedFlow targetFlow, FlowUpdateFsm stateMachine) {
-        RequestedFlow originalFlow = RequestedFlowMapper.INSTANCE.toRequestedFlow(flow);
-        stateMachine.setOriginalFlow(originalFlow);
+        stateMachine.setOriginalFlow(RequestedFlowMapper.INSTANCE.copy(flow));
```

Comment:
Need to check that this change will not return the already resolved bugs associated with multi table.

Reply:
Fixed